src/routes/+page.svelte

```diff
@@ -84,5 +89,8 @@
         {/if}
       </div>
     {/if}
+    {#if !loading && !$sessionStore.connectedStatus}
+      <div class="absolute right-0 bottom-0 left-0 text-center bg-base-300 text-sm py-2 font-monospace">Presets will only be saved to your local file system until you register</div>
```

Comment:
```suggestion
      <div class="absolute right-0 bottom-0 left-0 text-center bg-base-300 text-sm py-2 font-monospace">Presets will only be saved to your locally until you connect with the Ditto companion app</div>
```

Fits the context here where we ask users to connect to the web app.